src/main/java/io/cdap/plugin/ariba/source/connector/AribaConnector.java

```diff
@@ -101,6 +101,7 @@ public AribaConnector(AribaConnectorConfig config) {
   public void test(ConnectorContext connectorContext) throws ValidationException {
     FailureCollector collector = connectorContext.getFailureCollector();
     config.validateCredentials(collector);
+    collector.getOrThrowException();
     config.validateToken(collector);
```

Comment:
```suggestion
    FailureCollector collector = connectorContext.getFailureCollector();
    config.validateCredentials(collector);
    config.validateToken(collector);
    collector.getOrThrowException();
```

Reply:
Need to throw an exception from validateCredentials method before validating the token, if the fields are null. Then what would be the best way to handle it?

Reply:
```suggestion
    FailureCollector collector = connectorContext.getFailureCollector();
    config.validateCredentials(collector);
    collector.getOrThrowException();
    config.validateToken(collector);
    collector.getOrThrowException();
```

Reply:
Done